docker-compose-test.yml

```diff
@@ -73,19 +72,21 @@ services:
         aliases:
           - logstash.local
   vulnwhisperer:
-    image: hasecuritysolutions/vulnwhisperer:latest
+    # image: hasecuritysolutions/vulnwhisperer:latest
+    image: vulnwhisperer-local
     container_name: vulnwhisperer
     entrypoint: [
                   "vuln_whisperer",
+                  "-F",
                   "-c",
                   "/opt/VulnWhisperer/vulnwhisperer.ini",
                   "--mock",
                   "--mock_dir",
                   "/tests/data"
     ]    
     volumes:
-      # - /opt/VulnWhisperer/data/:/opt/VulnWhisperer/data
-      - ./docker_data/data/:/opt/VulnWhisperer/data
+      - ./data/vulnwhisperer/:/opt/VulnWhisperer/data
```

Comment:
I understand all of the folders under `.data` (vulnwhisperer, esdata1 and es_snapshots) are first created by the mocking tests locally and then mapped on the containers? I don't know where the folders are created and filled with the data.

Reply:
Nevermind, I saw them being created here https://travis-ci.org/pemontto/VulnWhisperer/builds/520628118